src/themes/teams-dark/components/Avatar/avatarVariables.ts

```diff
@@ -0,0 +1,9 @@
+export interface IAvatarVariables {
```

Comment:
the type declared for default theme should be reused - this would guarantee consistency of variables across different theme variants